packages/client/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - re-export `getSs58AddressInfo` from `@polkadot-api/substrate-bindings`
 
+### Changed
+
+- `polkadot-signer`: Add API to sign raw-data and rename `sign` to `signTx`
```

Comment:
This should be under breaking as well?

Reply:
`Changed` -> `Breaking`

same thing, really... but `Changed` is [the CHANGELOG TM name](https://keepachangelog.com/en/1.1.0/#how) 😉